Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.h

```diff
@@ -718,6 +718,12 @@ class UnifiedPDFPlugin final : public PDFPluginBase, public WebCore::GraphicsLay
     static constexpr double minimumZoomScale = 0.2;
 #endif
     static constexpr double maximumZoomScale = 6.0;
+
+#if PLATFORM(MAC)
+    static constexpr bool hasFullAnnotationSupport = false;
+#else
+    static constexpr bool hasFullAnnotationSupport = false;
```

Comment:
Is one of these incorrect? (Or should the platform define be removed)

Reply:
Indeed, PLATFORM(MAC) case is incorrect